contracts/GameJam.sol

```diff
@@ -23,4 +54,26 @@ contract GameJam {
 
     emit CompetitorAdded(competitor);
```

Comment:
should the `addCompetitor` function be limited to a stage? To avoid adding competitors after finish

Reply:
Yeah, it should!

Reply:
`addCompetitor` is now limited to the `Registration` Stage